docs/src/layouts/AppHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import NextLink from 'next/link';
+import { styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import SvgMuiLogo from 'docs/src/icons/SvgMuiLogo';
+import HeaderNavBar from 'docs/src/components/HeaderNavBar';
+import HeaderNavDropdown from 'docs/src/components/HeaderNavDropdown';
+
+const Header = styled('div')(({ theme }) => ({
```

Comment:
On the [Webflow demo](https://rebranding-demo.webflow.io/), I have used the same max-width on the navbar as in the hero container (on the demo I used 1200px as max-width). 

I don't have strong opinions about it, I did the change yesterday after seeing that most of the sites I'm using as a benchmark do that as well (Stripe, Linear, Tailwind, Watershed, Basecamp, etc.) It looks good but not using looks good as well. What do you think?

Reply:
I think we can go with the default maxWidth and refine at the end because it is low priority.